src/Mantasflowers.Persistence/Authentication/FirebaseContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿using FirebaseAdmin.Auth;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Mantasflowers.Persistence.Authentication
+{
+    public sealed class FirebaseContext
+    {
+        public Task SetCustomUserClaimsAsync(string uid, Dictionary<string, object> claims)
```

Comment:
Maybe this could accept actual GUID instead of string and then do the conversion by itself?